main.js

```diff
@@ -0,0 +1,87 @@
+// Create a `tree` function that should build a pine tree out of a character in the Chrome dev tools console. 
+
+// ### It accepts a single object as an argument. The object should have two key/value pairs.
+
+// 1. A key that specifies the height of the pine tree.
+// 1. The value for the height of the tree should be from user input in a `<input type="text">` field in the DOM.
+// 1. A key that specifies which character to use to build the pine tree.
+// 1. The character to use should be from user input in a `<input type="text">` field in the DOM.
+
+// Once the user enters in a number, and a character, the user can either then just press the 
+// enter key _(as long as the cursor is in one of the input fields)_, 
+// or click a button that is labeled "Grow your tree" and the tree should be shown in the console. 
+// This requires you to add an event listener to the button, as well as an event listener for the enter/return key.
+
+// If either of the input fields does not have a value in it when the user presses the enter key, or presses the button, then display an alert stating that both fields must have a value.
+ 	
+ // Get user data
+ 	var treeChar = document.getElementById("character").value;
+	var treeHeight = document.getElementById("height").value;
+
+	var button = document.getElementById("myButton");
+	var treeSpace = "";
+	var tree = "";
+
+	var treeArray = {
+  		height: treeHeight.value,
+  		character: treeChar.value
+};
+  		
+
+	button.addEventListener("click", userInput);
```

Comment:
So, calling this function in the way that you are calling it is not 100% necessary, since in the next line you have a event on the same 'click'. Try refactoring your code by calling the userInput function with the event listener only, and then call your growTree if they have entered a value for both of the fields.